Traceback (most recent call last):
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\RUN.py", line 1369, in <module>
    main()
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\RUN.py", line 1100, in main
    button_map_creator.tick(screen, mouse_pos, mouse_single_tick, glitch)
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\button.py", line 134, in tick
    return self.action(arg) if arg != None else self.action(self.args)
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\RUN.py", line 352, in launch_map_editor
    map_creator.launch()
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\map_creator.py", line 570, in launch
    me.tick()
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\map_creator.py", line 559, in tick
    self.tick_editor()
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\map_creator.py", line 525, in tick_editor
    self.exportButton.tick(self.screen, self.mouse_pos, self.mouse_single_tick, None)
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\button.py", line 134, in tick
    return self.action(arg) if arg != None else self.action(self.args)
  File "c:\Users\Reset\Documents\GitHub\OVRDOZE\map_creator.py", line 123, in exportLevel
    with open(open(pLevel + f"/walls.txt"), "w") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Reset\\Documents\\GitHub\\OVRDOZE/ovrdoze_data/levels/MyLevel/walls.txt'
